inc/namespace.php

```diff
@@ -29,7 +29,7 @@ function bootstrap() {
 	}
 
 	add_filter( 'wp_mail_from', function ( string $email ) use ( $config ) : string {
-		return $config['email-from-address'];
+		return ! empty( $config['email-from-address'] ) ? $config['email-from-address'] : $email;
```

Comment:
We could probably validate this too with a filter function like:
```suggestion
		return filter_var(
			$config['email-from-address'],
			FILTER_VALIDATE_EMAIL,
			FILTER_NULL_ON_FAILURE
		) ?? $email;
```